composer.json

```diff
@@ -420,7 +415,9 @@
             }
         },
         "drupal-lenient": {
-            "allowed-list": ["drupal/ckeditor_acr"]
+            "allowed-list": [
+                "drupal/ckeditor_acr"
```

Comment:
I believe this `drupal-lenient` block can be removed and the `mglaman/composer-drupal-lenient` plugin be uninstalled, since we don't need it for `ckeditor_acr` anymore. It allowed installing modules that had a lower drupal version dependency than the one currently installed, but we don't need it for CKEditor 4 anymore.

Reply:
Excellent catch! I also removed the module "mglaman/composer-drupal-lenient" as ckeditor_acr appears to be the only one requires it.